docs/rule/require-valid-form-groups.md

```diff
@@ -0,0 +1,51 @@
+# require-valid-form-groups
+
+This rule requires appropriate semantics for grouped form controls. Correctly grouped form controls will take one of two approaches:
+
+* use `<fieldset>` + `<legend>` (preferred)
+* associate controls using WAI-ARIA (also acceptable)
+
+## Examples
+
+This rule **forbids** the following:
+
+```hbs
+<div>
+  <label for="radio-001">Chicago Zoey</label>
+  <input id="radio-001" type="radio" name="prefMascot-Zoey" value="chicago zoey" />
+  <label for="radio-002">Office Hours Tomster</label>
+  <input id="radio-002" type="radio" name="prefMascot-OfficeHoursTomster" value="office hours tomster" />
+  <label for="radio-003">A11y Zoey</label>
+  <input id="radio-003" type="radio" name="prefMascot-Zoey" value="a11y zoey" />
+</div>
```

Comment:
Could these test case be simplified a bit? Or could you add a comment inside pointing to the part that is incorrect? it's a bit hard to tell what's wrong at a glance.